examples/profile/dynamic-share.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html class="yoti-html">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <title>Yoti Dynamic Share example</title>
+  <link rel="stylesheet" type="text/css" href="/static/index.css" />
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet" />
+</head>
+
+<body class="yoti-body">
+
+  <main>
+    <section class="yoti-top-section">
+      <div class="yoti-logo-section">
+        <img 
+          class="yoti-logo-image"
+          src="/static/assets/logo.png" 
+          srcset="/static/assets/logo@2x.png 2x"
+          alt="Yoti"/>
+      </div>
+
+      <h1 class="yoti-top-header">We now accept Yoti</h1>
```

Comment:
Could we update this to be "Dynamic Share Example" too? (I know it's the page title. Just would be useful to have more of a distinction between the two pages 👍 
```suggestion
      <h1 class="yoti-top-header">Dynamic Share Example</h1>
```